src/ngraph/op/fused/group_conv.cpp

```diff
@@ -229,3 +229,241 @@ bool ngraph::op::GroupConvolution::has_groups_in_filters_shape() const
     // dim.
     return ((get_input_shape(0).size() + 1) == get_input_shape(1).size());
 }
+
+constexpr NodeTypeInfo op::GroupConvolutionBackpropData::type_info;
+
+op::GroupConvolutionBackpropData::GroupConvolutionBackpropData(
+    const Output<Node>& data_batch,
+    const Output<Node>& filters,
+    const Output<Node>& output_delta,
+    const Strides& window_movement_strides,
+    const Strides& window_dilation_strides,
+    const CoordinateDiff& padding_below,
+    const CoordinateDiff& padding_above,
+    const size_t groups)
+    : FusedOp({data_batch, filters, output_delta})
+    , m_window_movement_strides(window_movement_strides)
+    , m_window_dilation_strides(window_dilation_strides)
+    , m_padding_below(padding_below)
+    , m_padding_above(padding_above)
+    , m_groups(groups)
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::GroupConvolutionBackpropData::pre_validate_and_infer_types()
+{
+    element::Type data_element_type = get_input_element_type(0);
+    PartialShape data_pshape = get_input_partial_shape(0);
+    PartialShape filters_pshape = get_input_partial_shape(1);
+    PartialShape delta_pshape = get_input_partial_shape(2);
+
+    NODE_VALIDATION_CHECK(this,
+                          data_element_type.is_dynamic() || data_element_type.is_real(),
+                          "Argument element type must be f16, bf16, f32, f64 or dynamic (got ",
+                          data_element_type,
+                          ").");
+
+    if (data_pshape.is_dynamic() || filters_pshape.is_dynamic() || delta_pshape.is_dynamic())
+    {
+        set_output_type(0, data_element_type, PartialShape::dynamic());
+    }
+}
+
+size_t ngraph::op::GroupConvolutionBackpropData::get_groups() const
+{
+    NODE_VALIDATION_CHECK(this,
+                          m_groups.is_static(),
+                          "get_groups() can only be called if the number of groups is static.");
```

Comment:
There is no option to initialize groups by other type than `size_t` so this check is unnecessary.
Groups are a required `size_t` attribute for constructor and public setter is unavailable so `m_groups` will be always static.

Reply:
updated